tests/tests.schema

```diff
@@ -689,7 +692,10 @@
       }
     },
     "Microsoft.AgeEntityRecognizer": {
-      "$role": "implements(Microsoft.IEntityRecognizer)",
+      "$role": [
```

Comment:
I'm still coming up to speed on some of this schema functionality.  Are these schmea changes just additive and backwards compatible?  If this is merged, and I open a project created with 4.10 in 4.11, everything new just shows up without issue?

Reply:
yes additive